torch/csrc/autograd/saved_variable.cpp

```diff
@@ -55,6 +51,24 @@ Variable SavedVariable::unpack(std::shared_ptr<Function> saved_for) const {
     grad_fn = std::move(saved_for);
   }
 
+  if (saved_version_ != version_counter_.current_version()) {
+    std::stringstream message;
+    message << "one of the variables needed for gradient computation has been "
+        "modified by an inplace operation: [" << data_.type().toString() << " "
+        << data_.sizes() << "]";
+    if (grad_fn) {
+        message << ", which is output " << output_nr_
+            << " of " << grad_fn->name() << ",";
+    }
+    message << " is at version " << version_counter_.current_version()
+        << ", not expected version " << saved_version_ << ".";
```

Comment:
nit: I don't think you want the "not" here.

Reply:
It was on purpose, but I can see how it's ambiguous, thanks. I meant to say "it's at version x, not at the expected version, which is y". I'll change it to "is at version x, expected y instead".